lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -69,11 +78,14 @@ EOS
 sub execute {
     my $self = shift;
     
+    my $input_bam = $self->get_bam();
```

Comment:
Interesting.  I didn't realize that.  RIght now, it is conveniently only called once.  Same for the other output parameters `$self->pdf_file` and `$self->raw_stats_file`.